test/integration/visitor_can_view_cars_test.rb

```diff
@@ -13,17 +13,16 @@ class VisitorCanViewCarsTest < ActionDispatch::IntegrationTest
       assert page.has_content?('Cars')
     end
 
-    within("##{Car.first.id}") do
-      assert page.has_content?('1960 Chevy0 El Camino0')
-      assert page.has_content?('$100')
-      assert page.has_content?("Dave's Cars")
-    end
+    assert page.has_content?('1960 Chevy0 El Camino0')
+    assert page.has_content?('$100')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.